symfony/src/Controller/ArticleController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use Algolia\SearchBundle\IndexManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class ArticleController
+{
+
+    protected $indexManager;
+
+    public function __construct(IndexManagerInterface $indexingManager)
+    {
+        $this->indexManager = $indexingManager;
+    }
+
+    public function index()
+    {
+        $articles = $this->indexManager->rawSearch('', Article::class);
+        var_dump($articles);
```

Comment:
You should return a json response just like you have done on laravel playground.

Reply:
`The controller must return a "Symfony\Component\HttpFoundation\Response" object`
This why i did this. Can you help me on what i should return ?

Reply:
I use JsonReponse from HttpFondation